### DISCUSSION TITLE
Need help Recover H96 Max X3

### DISCUSSION THREAD

**Discussion by diktus** [ophub/amlogic-s9xxx-armbian#2631] [Amlogic]
Hi,

My STB run latest Armbian_24.11.0_amlogic_s905x3_noble_6.6.58_server_2024.11.01.img.gz. it run on microsd with change the dtb file location in uEnv. But i made mistake with remove partition on eMMC and format it to BTRFS. Now my box is just bootlop.

I try to recover using USB Burning Tool with oem Firmware but it always end with romcode error. I Try using TWRP with boot maker it can enter TWRP. i need to press the toothpick on AV jack. When i try to install the oem Firmware it say that emmc is 0mb. I try to change the file system with advance wipe option but got error that the storage can not mount. I guess the TWRP not recognize the BTRFS. 

Is there any way to boot using microsd with something like TWRP Recovery but can recognize the eMMC?

**Comment by petesimon** [comment]
hello @diktus i purhcased this tv box in October 2024. follow my [guide](https://forum.armbian.com/topic/32488-h96-max-x3-armbian-install/?do=findComment&comment=205379) to boot TWRP recovery, and install stock android PPR1.180610.011.20220516.  (but note: stock android 20230208 does not permit booting armbian). afterwards you can install armbian.
https://forum.armbian.com/topic/32488-h96-max-x3-armbian-install/?do=findComment&comment=205379

*edit: another method is to use Burn Card Maker 2.0.3 and the file "AML_user_H96MaxX3_9_20220516.1821.img" from [this link](https://drive.google.com/drive/folders/1Rn7uFZ3SuR9OhMPrFkYMKU4sV1MXcBej) or [this link](https://my.hidrive.com/share/cvdmbc7ro2) to make the tv box update itself and install stock android. i [did that method](https://forum.armbian.com/topic/46910-unbrick-amlogic-s905x3-t95-max-plus-tv-box-after-install-armbian-tutorial/?do=findComment&comment=205571) for another similar s905x3 tv box

**Reply by diktus** [reply]
Hi,

thanks for the reply. I have try use TWRP but its show 0MB for the internal storage. Also try to wipe partition but TWRP can not mount and repartition. I think TWRP seems does not recognize the BTRFS partition.

I try the method using burning card and boot with hold toothpick. I wait quite long nothing happens maybe i will try to wait longer

**Reply by petesimon** [reply]
hello. you're welcome. these low-cost tv boxes are interesting to me because i can use one for NAS and another for a web-service or app such as Adguard or Jellyfin. Anyways, are you sure the TWRP you are booting is 100% compatible with your tv box? - Did you download [my copy of TWRP](https://my.hidrive.com/share/cvdmbc7ro2) and try it?

Here are some more tips that i found online: (1) try using a USB flash drive (or card reader adapter) in the USB 2.0 port or USB 3.0 port instead of the TF slot for cards; (2) try different versions of Burn Card Maker and USB Burn Tool; (3) try different classes or speeds of the micro sd card and USB flash drives. Be sure to browse the official armbian forums for similar issues, re-read [the FAQ there](https://forum.armbian.com/forum/189-faq/) and re-read my two posts above.

**Reply by diktus** [reply]
Yes i follow your post before and download the TWRP. The version is 3.3. Do you think we can make armbian to boot like TWRP but with more tool like fdisk and mk4fs

**Reply by petesimon** [reply]
> Do you think we can make armbian to boot like TWRP but with more tool like fdisk and mk4fs

i don't know
